backend/src/packages/auth/auth.service.ts

```diff
@@ -87,6 +87,21 @@ class AuthService {
 
     return user;
   }
+
+  public async deleteAuthenticatedUser(
```

Comment:
I think auth.service is not responsible for deleting user

Reply:
I placed the delete account functionality in the auth controller and service because to me it seemed authentication related (create an account, delete account), since deleting an account often requires the user to be authenticated, it makes sense for the delete functionality to be a part of the AuthService as this service typically handles authenticated actions.

Also in a commercial app account deletion often requires additional security checks like password revalidation. Since AuthService is handling other security-related operations, it might be good to leave it there for future changes in the functionality.

That's what I thought but if it needs to be moved it can be done, Do you want me to leave it in auth controller or do you want me to create a user controller? or maybe move it elsewhere, let me know and I will work on it.

Reply:
@v0ldemar01

Reply:
I read your arguments, but let's handle it in user controller